### PR TITLE
mcs: forward ask split request

### PR DESCRIPTION
**Pull request by rleungx**
### What problem does this PR solve?

Issue Number: Ref #5839.

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Manual test

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by JmPotato — APPROVED** [review]

File: client/go.mod [hunk 1]
```diff
@@ -41,3 +41,5 @@ require (
 	gopkg.in/natefinch/lumberjack.v2 v2.2.1 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/pingcap/kvproto => github.com/rleungx/kvproto v0.0.0-20231017074040-7aea7783b691
```
Comment (lhy1024) [inline client/go.mod]: need update kvproto

File: pkg/mcs/scheduling/server/grpc_service.go [hunk 2]
```diff
@@ -254,6 +257,74 @@ func (s *Service) GetOperator(ctx context.Context, request *schedulingpb.GetOper
 	}, nil
 }
 
+// AskBatchSplit implements gRPC PDServer.
+func (s *Service) AskBatchSplit(ctx context.Context, request *schedulingpb.AskBatchSplitRequest) (*schedulingpb.AskBatchSplitResponse, error) {
```
Comment (lhy1024) [inline pkg/mcs/scheduling/server/grpc_service.go:261]: It seems to be similar to `HandleAskBatchSplit`, could we reduce it?

**Review by lhy1024 — APPROVED** [review]

File: pkg/mcs/scheduling/server/grpc_service.go (hunk 2 above)
Comment (rleungx) [reply pkg/mcs/scheduling/server/grpc_service.go:261]: What do you mean?

**Pull request merged**